libs/coin-framework/src/mocks/account.ts

```diff
@@ -417,18 +416,10 @@ export function genAccount(
         opts.tokenIds?.includes(id)
     );
     const tokensFromOpts = all.filter((t) => opts.tokenIds?.includes(t.id));
-    const compoundReadyTokens = all.filter(findCompoundToken);
-    const notCompoundReadyTokens = all.filter((a) => !findCompoundToken(a));
 
     // [ explicit token from opts ] > compoundReady > rest
     const tokens = tokensFromOpts
-      .concat(compoundReadyTokens)
-      .concat(
-        // from random index
-        notCompoundReadyTokens.slice(
-          rng.nextInt(Math.floor(notCompoundReadyTokens.length / 2))
-        )
-      )
+      .concat(all)
```

Comment:
Do we care about having duplicates in the tokens array ?
If I understand the logic in here, it looks to me that we can have the same token twice because of the concat and this was probably also present before this change

Reply:
indeed it sounds like a bug in this mock generator (cc @haammar-ledger )
bug that i have "preserved" =) i mean it was here before, it still is